build.py

```diff
@@ -38,6 +38,7 @@
 use_plugin("source_distribution")
 
 use_plugin("python.unittest")
+use_plugin("python.pytest")
```

Comment:
Hi, guys.
Do You have any ideas how we can test pytest plugin without `build_depends_on('pytest')` usage or `use_plugin` + `plugin_depends_on('pytest')`?